Bottom: 101bcf45ed8274f84c847c617f234f2230a1ecac
Top:    b10065cf867024206d20aa19ed40ec83fca1b64e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 19:41:14 +0000

Refresh of work-in-progress-on-arc

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 3d62222..8c62938 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -454,7 +454,7 @@ new_descriptor (VNODE * a, char poly, char side)
 }
 
 /*
- * Compare the edge angles (and ourvatures) to determine
+ * Compare the edge angles (and curvatures) to determine
  * the ordering of two edges around a vertex.
  *
  * Returns <0 (ie -1) for a < b
